internal/msalbase/internal.go

```diff
@@ -18,7 +18,6 @@ import (
 	"github.com/AzureAD/microsoft-authentication-library-for-go/internal/json"
 	jwt "github.com/dgrijalva/jwt-go"
 	uuid "github.com/google/uuid"
-	"github.com/shirou/gopsutil/host"
```

Comment:
This looks like the only usage of this package, can we take it out of the go.mod/go.sum?

Reply:
We can, but we should do that with a go mod tidy in the future.

What do you think about you adding a check that the go.mod file is updated via tidy?